src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java

```diff
@@ -0,0 +1,83 @@
+package microsoft.exchange.webservices.data;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import javax.xml.stream.XMLStreamException;
+
+public class UniqueBodyTest {
+
+  UniqueBody impl;
+
+  final String text = "test";
+
+  final BodyType bodyType = BodyType.HTML;
+
+  @Mock EwsServiceXmlReader reader;
+  @Mock EwsServiceXmlWriter writer;
+
+
+  @Before public void setUp() throws Exception {
+
```

Comment:
There seems to be randomly blank lines at the beginning or end of functions.  There also sometimes double blank lines.  I would be consistent with no blank lines at top or bottom of functions and removing doubles blank lines (make them single).